### DISCUSSION TITLE
Noise

### DISCUSSION THREAD

**Discussion by drwebbe** [optimocha/speed-booster-pack#67] [Q&A]
Hi,
I added my own SVG icons to the elementor pages. When using Speedbooster, I get text before those icons : ___noise___ followed by a number...
Any idea where this is coming from ?

**Comment by optimocha** [comment]
@drwebbe I replied on WP.org but after some digging, I saw that the Simple HTML DOM library we're using has a method that adds `___noise___` plus some number while optimizing your pages' HTML source codes.

It's likely a plugin conflict is causing this trouble. Can you go to Tools > Site Health, then the click Info tab, then click the "Copy site info to clipboard" button and paste it here? You can redact any other parts except wp-mu-plugins and wp-plugins-active, which are what I need to see.
